Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteScrollingCoordinatorProxyIOS.mm

```diff
@@ -179,6 +179,22 @@
     sendUIStateChangedIfNecessary();
 }
 
+#if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
+void RemoteScrollingCoordinatorProxyIOS::scrollingTreeFixedNodesDidUpdate()
```

Comment:
When `scrollingTreeNodeDidScroll()` (which calls this) is called, we haven't updated all layer positions, so I'm not convinced that it's valid to check the bounds of CALayers here.

Reply:
Moved it to `applyScrollingTreeLayerPositionsAfterCommit`